apps/ledger-live-desktop/static/i18n/el/app.json

```diff
@@ -4229,6 +4229,9 @@
     "EthAppPleaseEnableContractData": {
       "title": "Please enable contract data in the Ethereum app settings"
     },
+    "VechainAppPleaseEnableContractDataAndMultiClause": {
+      "title": "Please enable contract data and multi-clause in the Vechain app settings"
```

Comment:
We have to do more tests about this one, I managed to send 2 transactions with both contract data and multi-clause set to 'No', maybe because I had them previously enabled

Reply:
checked, you are right, removed the 'multi-clause' part